### DISCUSSION TITLE
How to modify the name display of "paste ="

### DISCUSSION THREAD

**Discussion by peter-RenPer** [pilotmoon/PopClip-Extensions#1226] [Q&A] [answered]
I don't like to paste as plain text to display "Paste =" How to change "Paste =" to "Paste"!

**Comment by pilotmoon** [comment, marked as answer]
This snippet will do it:
```
#popclip
name: Paste
requirements: [paste]
javascript: popclip.pasteText(pasteboard.text);
```
*(The above block is an [extension snippet](https://forum.popclip.app/t/introducing-extension-snippets/309) — select it then click "Install Extension" in PopClip.)*

**Reply by peter-RenPer** [reply]
Great. Problem solved. Thanks.

**Reply by peter-RenPer** [reply]
How can I call the right-click menu "Convert Text to Simplified Chinese in Service" on Mac
